src/useCroods.tsx

```diff
@@ -74,12 +74,11 @@ const useCroods = ({
   )
 
   const save = useCallback(
-    contextOpts => async ({
-      addToTop,
-      onProgress,
-      $_requestConfig = {},
-      ...rawBody
-    }: any) => {
+    ({
+      onProgress: onUploadProgress,
+      requestConfig = {},
+      ...contextOpts
+    }) => async ({ addToTop, ...rawBody }: any) => {
```

Comment:
Why not pass addToTop too to the contextOpts?

Reply:
I though about it, but wandered if there was a semantic reason to be there in the first place.. But I guess it makes sense to move it

Reply:
not sure why... this is some legacy that we brought and never used as far as I was on it.
But probably left it explicitly due to lack of documentation.